tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -20,6 +20,10 @@ def canvas_file_basic_lti_launch_caller(context, pyramid_request):
     BasicLTILaunchViews.canvas_file_basic_lti_launch(), and return whatever
     BasicLTILaunchViews.canvas_file_basic_lti_launch() returns.
     """
+    # The custom_canvas_course_id param is always present when
+    # canvas_file_basic_lti_launch() is called: Canvas always includes this
+    # param because we request it in our config.xml.
```

Comment:
This assumes the integrator hasn't modified the custom fields in the LTI configuration. Very unlikely, but technically possible to do.